src/style.css

```diff
@@ -0,0 +1,109 @@
+:root {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
```

Comment:
leave only variables inside `root`

move common font-styles to `html` selector